implementations/rust/ockam/ockam_command/tests/commands.bats

```diff
@@ -382,6 +429,37 @@ teardown() {
   assert_success
 }
 
+@test "inlet/outlet example with credentials 2/2" {
+  skip_if_orchestrator_tests_not_enabled
+
+  $OCKAM project info --name default --output json  > /tmp/project.json
+
+  run $OCKAM node create green --project /tmp/project.json --enable-credential-checks --no-shared-identity
+  assert_success
+  green_identifer=$($OCKAM identity show -n green)
+
+  run $OCKAM node create blue --project /tmp/project.json --enable-credential-checks --no-shared-identity
+  assert_success
+  blue_identifer=$($OCKAM identity show -n blue)
+
+  run $OCKAM project enroll --member $blue_identifer --to /project/default/service/authenticator
+  assert_success
+  run $OCKAM project enroll --member $green_identifer --to /project/default/service/authenticator
+  assert_success
+
+  run $OCKAM tcp-outlet create --at /node/blue --from /service/outlet --to 127.0.0.1:5000 --check-credential
+  assert_success
+  run  $OCKAM forwarder create blue --at /project/default --to /node/blue
+  assert_output --partial "forward_to_blue"
+  assert_success
+
+  run $OCKAM tcp-inlet create --at /node/green --from 127.0.0.1:7000 --to /project/default/service/forward_to_blue/secure/api/service/outlet --check-credential
```

Comment:
Would be great to test that whole session recovery is working. Unit/integration/bats tests would be great. Not sure how hard it would be to do. Not in this PR for sure